integrations/torchvision/tutorials/sparse-transfer-learning.md

```diff
@@ -0,0 +1,168 @@
+
+This page explains how to fine-tune a pre-sparsified Torchvision image classification models like ResNet-50 with SparseML's CLI.
+
+## Overview
+
+Sparse transfer learning is quite similiar to the typical transfer learning training, where we fine-tune a checkpoint pretrained on ImageNet onto a smaller downstream dataset. However, with Sparse Transfer Learning, we simply start the fine-tuning process from a pre-sparsified checkpoint and maintain sparsity while the training process occurs.
+
+[SparseZoo](https://sparsezoo.neuralmagic.com/?domain=cv&sub_domain=classification&page=1) contains pre-sparsified checkpoints for common image classification models like ResNet-50 and EfficientNet, which can be used as the starting checkpoints for the training process.
+
+## Installation
+
+Install SparseML via `pip`.
+
+```bash
+pip install sparseml[torchvision]
+```
+
+## Sparse Transfer Learning with ResNet-50
+
+Let's try a simple example of Sparse Transfer Learning ResNet-50 onto [FastAI's Imagenette dataset](https://github.com/fastai/imagenette) (a subset of 10 easily classified classes from ImageNet).
+
+Download Imagnette to your local directory:
+```bash
+wget https://s3.amazonaws.com/fast-ai-imageclas/imagenette2-320.tgz
+tar -xvf imagenette2-320.tgz
+```
+
+Kick off transfer learning.
+```bash
+sparseml.image_classification.train \
+    --recipe zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned95_quant-none?recipe_type=transfer-classification \
+    --checkpoint-path zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned95_quant-none?recipe_type=transfer-classification \
+    --arch-key resnet50 \
+    --dataset-path ./imagenette2-320\
+    --batch-size 32
+```
+
+Let's discuss the key arguments:
+- `--checkpoint-path` specifies the starting model to use in the training process. It can either be a local path to a PyTorch checkpoint or a SparseZoo stub. Here, we passed a stub that identifies a 95% pruned-quantized version of ResNet-50 in the SparseZoo. The script downloads the PyTorch model to begin training.
+
+- `--arch-key` specifies the torchvision architecture of the checkpoint. For example, `resnet50` or `mobilenet`.
+
+- `--dataset-path` specifies the dataset used for training. We passed the local path to Imagenette. The dataset should be in the ImageFolder format (we will describe the format below).
+
+- `--recipe` specifies the sparsity related parameters of the training process. Recipes are YAML files that declare the sparsity related algorithms that SparseML should apply. For transfer learning, the recipe instructs SparseML to maintain sparsity during training and to apply quantization over the final epochs. In this case, we passed a SparseZoo stub, which instructs SparseML to download a premade ResNet-50 transfer learning recipe. In addition to SparseZoo stubs, you can also pass a local path to a YAML recipe. See below for more details on what the transfer learning recipe looks like.
```

Comment:
sparsity related -> sparsity-related

Reply:
In this case, we passed a SparseZoo stub, instructing SparseML to download a premade ResNet-50 transfer learning recipe.